### DISCUSSION TITLE
Final reflections / 4th project iteration / links to sites and repositories

### DISCUSSION THREAD

**Discussion by benmiller314** [benmiller314/dsam2023fall#17] [Show and tell]
On this, our final discussion forum for the semester, please post the following by Wednesday, December 13 at 2pm:

1. A link to the final-for-now (a.k.a. Iteration 4) version of your **public-facing project**
    - As we discussed back at the previous presentation stage, please be sure to _give credit to sources_ you used within your public-facing site, e.g. by linking back to originals or by having a page within the project with more complete bibliographic information (title, author, location, license).
    - Note that you should also _credit yourself_! If your name and a brief bio do not yet exist on your public-facing site, consider: you use information about site creators to gauge your trust levels, right? Help others to trust you, too. : )
    - Please double-check that I have access to the site?
2. If relevant, a link to a **repository** of source materials that you have been shaping or processing to produce your project.
    - NB: Both GitHub or OneDrive offer functionality to share a folder, either publicly or with specific people. If you have a repo and aren't sure how to share it, I can help.
3. As much of your end-of-semester **reflection** as you're willing to share publicly. See [lesson 15](https://benmiller314.github.io/dsam2023fall/plans/week-15#finality) for guidelines.

**Comment by Rezaetmd** [comment]
Hi everyone!

When I was preparing for the first iteration of my project, for which I knew I’m going to take a look at some magazine archive, I didn’t know that it would reach a point in later iterations/phases that I should decide between two things: either develop a website on scalar and try to include as much as information as I can by the end of semester or skip modeling a website for most of the parts and engage with raw data pulled out of different sources. Knowing that Scalar is always there to work with but I would miss the chance to get feedback on how to work with data from my kind instructor Pro. Miller and amazing scholars in the class, I chose the latter one. And I enjoyed that! There were moments of frustrations but I guess that's how learning works. More than that I enjoyed having fruitful conversations with you all. I hope to see you in Practicum should I decide to take the course,

Best wishes and good luck with your amazing projects,
Reza 

Here are the links to my:
1.	Scalar project: https://scalar.usc.edu/works/filmfarsi 
    Username: ree36@pitt.edu
    Password: vogQaf-wykgiw-9caxhu 

2.	Spreadsheet: https://docs.google.com/spreadsheets/d/16DnPL6z-y6_18T8suDtJS7jA_HuXgb7YSDau9FqGZ4k/edit?usp=sharing

**Comment by SidraNArshad** [comment]
https://scalar.usc.edu/works/dsam-project---blm-activists-/index

**Comment by amj169** [comment]
https://sites.google.com/d/1RljPXM-aoMi04aLCkWH1blNv2w9DQs9f/p/1kiBeMjB1CKYZ38avAUEljDdBl3hCtRGb/edit
